packages/runtime/src/runtime.ts

```diff
@@ -87,6 +88,10 @@ export class Runtime {
       throw new RuntimeError(RUNTIME_ERRORS.GENERAL.ACCOUNT_DOES_NOT_EXIST,
         { address: address, line: lineNumber });
     }
+    if (a.address !== address){
+      throw new RuntimeError(RUNTIME_ERRORS.GENERAL.ACCOUNT_DOES_NOT_EXIST,
+          { address: address, line: lineNumber, message: 'Account and address mismatch!' });
```

Comment:
`ACCOUNT_DOES_NOT_EXIST` error will not display your message (`Account and address mismatch`). We should create a new Error for this, to be more resonating with the message  "Account and address mismatch!" (maybe `ERR_ACCOUNT_ADDR_MISMATCH`